files/build_templates/docker_image_ctl.j2

```diff
@@ -474,7 +474,15 @@ start() {
     {%- endif %}
 
     if [ -z "$DEV" ]; then
-        NET="host"
+    {%- if docker_container_name == "database" %}
+        NET="bridge"
+        PORT_MAP="-p 127.0.0.1:6379:6379"
+    {%- elif docker_container_name == "snmp" %}
+    NET="bridge"
```

Comment:
I just tried image built by this PR yesterday, version: SONiC.master-15176.360116-6bb24c135. Looks like rsyslog inside snmp container cannot work well. I cannot find any syslog generated by snmp container (contains 'snmp#') in local host syslog file or our syslog server, and process `containercfgd` which is to dynamically generate rsyslog configuration file inside container doesn't startup. Not sure whether you have verified this, could you confirm that?

Reply:
@ycoheNvidia  could check if change like this is needed. https://github.com/sonic-net/sonic-buildimage/pull/4738